test/extended/builds/pipeline_jenkins_e2e.go

```diff
@@ -380,26 +380,26 @@ var _ = g.Describe("[sig-devex][Feature:Jenkins][Slow] Jenkins repos e2e openshi
 					if err != nil || !br.BuildSuccess {
 						exutil.DumpBuilds(oc)
 						exutil.DumpPodLogsStartingWith("nodejs", oc)
-						exutil.DumpBuildLogs("nodejs-mongodb-example", oc)
-						exutil.DumpDeploymentLogs("mongodb", 1, oc)
-						exutil.DumpDeploymentLogs("nodejs-mongodb-example", 1, oc)
+						exutil.DumpBuildLogs("nodejs-postgresql-example", oc)
```

Comment:
Noticed you used postresql here - was this intentional?

Reply:
Yes, part of this PR is to remove all dependencies on mongodb, which we deprecated in 4.6 and is not available at all on ARM.  As noted above, once this merges, we can then remove mongodb entirely from cluster-samples-operator.